demo/src/core/scroll-to-top-component.js

```diff
@@ -0,0 +1,48 @@
+import { css, html, LitElement } from 'lit';
+import { animations, theme } from '../theme/theme';
+
+/**
+ * A custom web component that provides a button to scroll to the top of the page.
+ *
+ * @element scroll-to-top-button
+ */
+export class ScrollToTopComponent extends LitElement {
+  static styles = [
+    theme,
+    animations,
+    css`
+      .scroll-to-top-button {
+        position: fixed;
+        right: 20px;
+        bottom: 20px;
+        z-index: 1;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        width: var(--size-7);
+        height: var(--size-7);
+        padding: 0;
+        border: none;
+        border-radius: var(--radius-round);
+      }
+
+      .scroll-to-top-button i {
+        font-size: var(--size-8);
+      }
+  `];
+
+  constructor() {
+    super();
+  }
```

Comment:
@jboix Could we delete these lines, since we don't do anything with the `constructor`?

Reply:
Removed.